torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -181,6 +181,11 @@ def is_tensor(target):
 
     # I'd love to use a match statement here, but it wasn't introduced until py3.10
     if node.op == "call_function":
+        if node.meta and node.meta.get("is_wrapped", False):
+            # This is fx.wrap function
+            if node.meta.get("user_cache_hash", None):
```

Comment:
Don't we need this hash in the cache key even if it exists? I.e. in AOTAutogradCacheDetails

Reply:
yeah - it seems like if we are marking this node as safe in this function, we need some logic later on that actually grabs this hash and includes it in the cache key

Reply:
The hash logic I put in `torch/_inductor/codecache.py FxGraphHashDetails`, where I exctract `user_cache_hash` into a member field. (Node meta does not participate in hash).

Reply:
> The hash logic I put in torch/_inductor/codecache.py FxGraphHashDetails, where I exctract user_cache_hash into a member field.

are there changes in `codecache.py` to review? I don't see them in the PR

Reply:
oh I guess they get automatically pulled into the cache key when you add them as extra properties here: https://github.com/pytorch/pytorch/pull/150032/files#r2064904802